topics/gender/index.html

```diff
@@ -133,7 +133,8 @@ <h1 class="logotype">MIDAAS</h1></a>
           }
         },
         chart2: {
-          title: "Ratio of Male to Female Income by State",
+          title: "Ratio of Female to Male Income by State",
+          <p class="description">"This Map Shows the Difference Between Women and Men at Various Percentiles (Darker Colors Indicate a Higher Difference)",
```

Comment:
Let's take this line out for now. As it stands, this code would break the JSON syntax.